### PR TITLE
chore: Changing everything that ends with `postgre` to `postgres` in Spanner samples and tests

### PR DESCRIPTION
**Pull request by anuragsrivstv**

### CODE REVIEW

**Comment by Rishabh-V** [comment]
It is completely fine for this time. However, going forward we should try to keep the commits small, so that they are easier to review and also to fix. 

Here it is just a naming change and we should be fine. But imagine if there is just one common feedback item that needs to be replicated to all the files then it becomes a challenge for both author and reviewer. 

I have reviewed partially and will try to finish it tomorrow morning. Thanks.

**Review by Rishabh-V — COMMENTED** [review]
This looks fine to me. Tests are passing as well. Please mark the PR ready for review and request review from Amanda.

File: spanner/api/Spanner.Samples.Tests/OrderNullsAsyncPostgresTest.cs [hunk 1]
```diff
@@ -19,26 +19,26 @@
...
     [Fact]
-    public async Task TestOrderNullsAsyncPostgre()
+    public async Task TestOrderNullsAsyncPostgres()
     {
         // Arrange.
         await CreateTableForOrderingAsync(_spannerFixture.ProjectId, _spannerFixture.InstanceId, _spannerFixture.PostgreSqlDatabaseId);
 
         // Act.
-        var result = await _sample.OrderNullsAsyncPostgre(_spannerFixture.ProjectId, _spannerFixture.InstanceId, _spannerFixture.PostgreSqlDatabaseId);
+        var result = await _sample.OrderNullsAsyncPostgres(_spannerFixture.ProjectId, _spannerFixture.InstanceId, _spannerFixture.PostgreSqlDatabaseId);
```
[the remark this answers is not in the document]
Comment (anuragsrivstv) [reply spanner/api/Spanner.Samples.Tests/OrderNullsAsyncPostgresTest.cs:41]: Thanks, for this PR I would like to keep changes only to rectify `Postgre` to `Postgres` .

**Comment by anuragsrivstv** [comment]
Hi @amanda-tarafa  , Please have a look at this PR and let me know what do you think. Thanks.

**Comment by amanda-tarafa** [comment]
If you change the name of the sample files, you need to update docs in google3 to reference the new files. We cannot merge this PR as is.
So:
- This PR should only add new files.
- You should then update docs to reference the new files.
- Only then you can delete the old files.

**Comment by Rishabh-V** [comment]
Thanks Amanda. I thought the same after seeing the snippet-bot check. I will work with Anurag on this and get it sorted.

**Comment by anuragsrivstv** [comment]
Okay, Thanks Amanda, I'll close this PR and raise another PR to only add new files.

**Pull request closed without merging**